tests/fixture_plugins/demultiplex_fixtures/sample_fixtures.py

```diff
@@ -120,16 +52,45 @@ def hiseq_2500_custom_index_bcl_convert_lims_samples(
 
 
 @pytest.fixture
-def lims_novaseq_samples_raw(lims_novaseq_samples_file: Path) -> list[dict]:
-    """Return a list of raw flow cell samples."""
-    return ReadFile.get_content_from_file(
-        file_format=FileFormat.JSON, file_path=lims_novaseq_samples_file
-    )
+def novaseq_6000_pre_1_5_kits_bcl2fastq_lims_samples(
+    novaseq_6000_pre_1_5_kits_flow_cell: Path,
+) -> list[FlowCellSampleBcl2Fastq]:
+    path = Path(novaseq_6000_pre_1_5_kits_flow_cell, "HLYWYDSXX_bcl2fastq_raw.json")
+    return [FlowCellSampleBcl2Fastq.model_validate(sample) for sample in read_json(path)]
+
+
+@pytest.fixture
+def novaseq_6000_pre_1_5_kits_bcl_convert_lims_samples(
+    novaseq_6000_pre_1_5_kits_flow_cell: Path,
+) -> list[FlowCellSampleBCLConvert]:
+    path = Path(novaseq_6000_pre_1_5_kits_flow_cell, "HLYWYDSXX_bcl_convert_raw.json")
+    return [FlowCellSampleBCLConvert.model_validate(sample) for sample in read_json(path)]
+
+
+@pytest.fixture
+def novaseq_6000_post_1_5_kits_bcl_convert_lims_samples(
+    novaseq_6000_post_1_5_kits_flow_cell: Path,
+) -> list[FlowCellSampleBCLConvert]:
+    path = Path(novaseq_6000_post_1_5_kits_flow_cell, "HK33MDRX3_bcl_convert_raw.json")
+    return [FlowCellSampleBCLConvert.model_validate(sample) for sample in read_json(path)]
+
+
+@pytest.fixture
+def novaseq_x_lims_samples(novaseq_x_flow_cell_directory: Path) -> list[FlowCellSampleBCLConvert]:
+    path = Path(novaseq_x_flow_cell_directory, "22F52TLT3_raw.json")
+    return [FlowCellSampleBCLConvert.model_validate(sample) for sample in read_json(path)]
+
+
+# Sample sheet creators
 
 
 @pytest.fixture
-def lims_novaseq_6000_sample_raw(lims_novaseq_6000_samples_file: Path) -> list[dict]:
-    """Return the list of raw samples from flow cell HVKJCDRXX."""
-    return ReadFile.get_content_from_file(
-        file_format=FileFormat.JSON, file_path=lims_novaseq_6000_samples_file
+def bcl_convert_sample_sheet_creator(
+    bcl_convert_flow_cell: FlowCellDirectoryData,
+    novaseq_x_lims_samples: list[FlowCellSampleBCLConvert],
+) -> SampleSheetCreatorBCLConvert:
+    """Returns a sample sheet creator for version 2 sample sheets with dragen format."""
```

Comment:
```suggestion
    """Returns a sample sheet creator for version 2 sample sheets with BCL Convert format."""
```

Reply:
Rewrote a bit as the converter is not actually needed, is implied from the sample sheet v2